add-column.md

```diff
@@ -18,6 +18,104 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
 
 ## Parameters
 
+| Parameter | Description |
+|-----------|-------------|
+| `table_name` | The name of the table to which you want to add the column. |
+| `name` | The name of the column you want to add. The name must be unique to the database and follow these [identifier rules](keywords-and-identifiers.html#identifiers).  |
+| `typename` | [Data type](data-types.html) of the new column. |
+| `col_qualification` | A list of column definitions, which may include [column-level constraint](constraints.html), [collation](collate.html), and/or [column family](column-families.html). |
+
 ## Examples
 
-## See Also
\ No newline at end of file
+### Add a Single Column
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN names STRING;
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+~~~
+
+
+### Add Multiple Columns
+
+~~~ sql
+> ALTER TABLE account ADD COLUMN location STRING, ADD COLUMN amount DECIMAL;
+~~~
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
+| location  | STRING            | true  | NULL    | {}        |
+| amount    | INT               | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+
+~~~
+
+### Add a Non Null Column with a Default Value
+
+~~~ sql
+> ALTER TABLE account ADD COLUMN amount DECIMAL NOT NULL DEFAULT (DECIMAL '1.3');
+~~~ 
+
+
+### Add a Non Null Column with Unique Values
+
+~~~ sql
+> ALTER TABLE account ADD COLUMN amount DECIMAL UNIQUE NOT NULL;
+~~~ 
+ 
+
+### Add a Column with Collation
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN names STRING COLLATE en;
+~~~ 
+ 
+
+### Add a Column to a New Column Family
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN location STRING CREATE FAMILY new_family;
+~~~ 
+
```

Comment:
Actually, I just noticed that you have three column family examples. It's your call, but it might be more efficient to strip down to a single example `Add a Column to a Column Family` and then use introductory sentences before each sub-example instead of code-block comments.